src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22594.cs

```diff
@@ -0,0 +1,25 @@
+﻿#if ANDROID
```

Comment:
Can we make this just run for all platforms? The bug is on Android, but lets make sure it doesn't happen on the other platforms as well